### PR TITLE
Fix send file for issue #8

### PR DESCRIPTION
**Pull request by rst5nn**
Multi-part form request is expected to be an array of JSON objects.

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ class RPClient {
                 content: fileData
             };
             rest.post(url, {
-                data: _self.buildMultiPartStream(saveLogRQ, file, boundary),
+                data: _self.buildMultiPartStream([saveLogRQ], file, boundary),
```
Comment (avarabyeu) [inline lib/client.js:100]: I believe, we should fix agents before merging it. Seems like they already put array here

**Review by avarabyeu — COMMENTED** [review]
Requires fix/check of existing agents

File: lib/client.js (hunk 1 above)
Comment (rst5nn) [reply lib/client.js:100]: Or as workaround rename `saveLogRQ` parameter to something like `arrSaveLogRQ`

**Comment by avarabyeu** [comment]
Rejected. Value is already supposed to be an array.

**Pull request closed without merging**